AdobeImsApi/Api/Data/UserProfileInterface.php

```diff
@@ -19,24 +19,24 @@ interface UserProfileInterface extends ExtensibleDataInterface
     /**
      * Get ID
      *
-     * @return int
+     * @return int|null
      */
     public function getId();
 
     /**
      * Get user ID
      *
-     * @return int
+     * @return int|null
      */
-    public function getUserId(): int;
+    public function getUserId(): ?int;
 
     /**
      * Set user ID
      *
-     * @param int $value
+     * @param int|null $value
      * @return void
      */
-    public function setUserId(int $value): void;
+    public function setUserId(?int $value): void;
```

Comment:
What is the use case for setting the user id to null?

Reply:
Because every field is nullable, there are should be a way to return fields or the entire object to an initial state.

Reply:
Decided to remove nullable setters for now because it's too debatable.